File: codekata/absolute beginner/779.js
/*
You are provided with two numbers. Find and print the smaller number.

Positive TestCase :
    Input-23 12
    Output-12
*/

function fun(x){
    var a=x.split(" ")
    if (a[0]>a[1]){
      console.log(a[1])
    }
    else{
      console.log(a[0])
    }
}

fun("23 12")

File: codekata/absolute beginner/779.py
'''You are provided with two numbers. Find and print the smaller number.

Input Description:
You are provided with two numbers as input.

Output Description:
Print the small number out of the two numbers.

Sample Input :
23 1
Sample Output :
1'''
num1,num2 = input().split()
num1,num2 = float(num1),float(num2)
if num1 > num2:
  print(round(num2))
else:
  print(round(num1))